pom.xml

```diff
@@ -350,5 +350,21 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <configuration>
+                    <rules>
+                        <bannedDependencies>
+                            <excludes combine.children="append">
```

Comment:
excludes -> includes

Reply:
Nope. Include is allow list
Exclude is ban list.